app/controllers/catalog_controller.rb

```diff
@@ -255,26 +255,10 @@ def resolve_ark
     redirect_to(solr_document_path(id: document.id))
   end
 
-  # Create an endpoint for PPPL / OSTI harvesting
-  # A catalog query for PPPL: https://datacommons.princeton.edu/discovery/?f%5Bcommunity_root_name_ssi%5D%5B%5D=Princeton+Plasma+Physics+Laboratory&q=&search_field=all_fields&fl=*&format=json
-  def pppl
-    # Limit to items from PPPL
-    pppl_query = "community_root_name_ssi:Princeton Plasma Physics Laboratory"
-
-    query = { q: pppl_query, fl: '*', format: 'json', sort: 'timestamp desc' }
-
-    solr_response = search_service.repository.search(**query)
-
-    @documents = solr_response.documents
-    respond_to do |format|
-      format.json { render json: @documents }
-    end
-  end
-
   # Create an endpoint for PPPL / OSTI harvesting that provides full datacite records
   def pppl_reporting_feed
     # Limit to items from PPPL
-    pppl_query = "community_root_name_ssi:Princeton Plasma Physics Laboratory"
+    pppl_query = "data_source_ssi:pdc_describe community_root_name_ssi:Princeton Plasma Physics Laboratory"
```

Comment:
I think the `community_root_name_ssi` filter should be in quotes, no?

Reply:
(I realize the filter didn't have quotes before, but without them the query will do a much wider search than just the matches in the `community_root_name_ssi` field)